sormas-ui/src/main/java/de/symeda/sormas/ui/utils/AbstractEditAllowedDetailView.java

```diff
@@ -0,0 +1,34 @@
+package de.symeda.sormas.ui.utils;
+
+import com.vaadin.ui.Component;
+import de.symeda.sormas.api.CoreFacade;
+import de.symeda.sormas.api.ReferenceDto;
+
+/**
+ * A detail view shows specific details of an object identified by the URL parameter.
+ *
+ * In addition to that, it contains generic code to check if the object is editable.
+ *
+ * @param <R>
+ *            {@link ReferenceDto} with the uuid as parsed from the URL.
+ */
+public abstract class AbstractEditAllowedDetailView<R extends ReferenceDto> extends AbstractDetailView<R> {
+
+	private CoreFacade coreFacade;
+
+	protected AbstractEditAllowedDetailView(String viewName, CoreFacade coreFacade) {
```

Comment:
This pattern caches the Facade instance which is discouraged because it does not refresh when the sormas-ear is redeployed.

Reply:
fixed